### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by shenvitor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+This repository is under active development.
+
+We are doing amplitude analysis / Partial Wave Analysis (PWA) with the use of symbolic expressions, which is included in the `ComPWA` packages, or CAS(Computer Algebra System)-assisted model building in general (the python package `SymPy` in this case).
+
+The first demonstration example is: $p \gamma \to$ Resonances (e.g. N\*) $\to \eta \pi^0 p$
+
+The main focus of this research would be: $p \gamma \to$ Resonances (e.g. N\*) $\to K^+ \pi^0 \Lambda$
+
+(More information to be added...)
```
Comment (redeboer) [inline README.md:11]: Better to remove or fill

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Research project on amplitude models for N-star and other resonances in the GlueX experiment
```
Comment (redeboer) [inline README.md:1]: baryon resonances (?)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+This repository is under active development.
+
+We are doing amplitude analysis / Partial Wave Analysis (PWA) with the use of symbolic expressions, which is included in the `ComPWA` packages, or CAS(Computer Algebra System)-assisted model building in general (the python package `SymPy` in this case).
```
Comment (redeboer) [inline README.md:5]: - ComPWA without backticks (code style)
- Same for SymPy, or write `sympy` (the package name)

**Review by redeboer — APPROVED** [review]

**Comment by redeboer** [comment]
In principle good, just left a few comments

File: README.md (hunk 2 above)
Comment (shenvitor) [reply README.md:1]: I think not only baryon resonances, in the later stage, meson resonances are also considered, e.g. K* resonances.

File: README.md (hunk 1 above)
Comment (shenvitor) [reply README.md:11]: Ok

**Pull request merged**